CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Ship a [zipapp](https://docs.python.org/3/library/zipapp.html) of pipx
 
 - Change the program name to `path/to/python -m pipx` when running as `python -m pipx`
+- Add hatch entry point config description to README
```

Comment:
The README is not a part of the actual package so I don’t think it should have a changelog (not entirely sure)

Reply:
Removed.